test/presubmit-tests.sh

```diff
@@ -116,7 +128,8 @@ kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo/$ARGO_V
 
 echo "submitting argo workflow for commit ${PULL_PULL_SHA}..."
 ARGO_WORKFLOW=`argo submit $(dirname $0)/${WORKFLOW_FILE} \
--p commit-sha="${PULL_PULL_SHA}" \
+-p image-build-context-gcs-uri="$remote_code_archive_uri" \
+-p target-image-prefix="${GCR_IMAGE_BASE_DIR}/" \
 -p test-results-gcs-dir="${TEST_RESULTS_GCS_DIR}" \
 -p cluster-type="${CLUSTER_TYPE}" \
 -p bootstrapper-image="${GCR_IMAGE_BASE_DIR}/bootstrapper" \
```

Comment:
consider passing suffix since you pass the base dir already above.

Reply:
This feature does not seem to be required for this PR which is already quite big (I'd have split it into three PRs, but it would take longer to get them in).